app/js/event/event.filter.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('upcomingStops.event')
+        .factory('eventFilter', eventFilter);
```

Comment:
In it's simplest form a factory should look like this: https://github.com/APT-project/upcoming/blob/master/app/js/event/event.factory.js

So the proximity function should be returned so that it can be called from outside.
It looks like you have commented those lines out // (10-20). You can replace them with the stuff you have written here.

Also we should probably rename the file to event.sort.js etc.